src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String toString() {
+        return "Circle{"
+                + "radius=" + radius
+                + "color=" + getColor()
+                + "}";
+    }
+
+    @Override
+    public void draw() {
```

Comment:
The 'draw' method is correctly overridden to print the Circle object, which aligns with the Drawable interface.